app/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,7 @@
     <string name="only_available_to_mentor">Only available to mentor</string>
     <string name="only_available_to_mentee">Only available to be a mentee</string>
     <string name="not_available_to_mentor_or_mentee">Not available to be a mentor or a mentee</string>
+    <string name="both_users_only_available_to_mentor">This request is not possible because both users are only ‘available to mentor'</string>
```

Comment:
Please add \ character before apostrophe (') character because in strings.xml file, apostrophe (') character is added after \ this character. That's why Travis build failed - **it showed unescaped apostrophe in string error**